blueprint/index.md

```diff
@@ -7,232 +7,285 @@ icon: blueprint
+
+## Solution
+Genesys Bot Connector provides the API to call third-party bots in an Architect message flow. The blueprint showcases how to connect to the Microsoft Power VA bot in Genesys Cloud using the AWS services:
+
+1. Create an application using AWS Lambda functions. The application known as the Bot Interpreter acts in between Genesys Cloud and Power VA. 
+2. Create a chatbot with topics and entities in Microsoft Power VA.
+3. Configure a Messenger in your website.
+
+The Lambda application receives the utterances from the Messenger through Genesys Bot Connector. The Lambda application acts in between Genesys Cloud and the Power VA. The application changes the format of the request before sending out an HTTP request to Power VA for Natural Language Understanding (NLU). It also converts the response message received from Power VA to match the postUtterance API that is provided by the Genesys Bot Connector.
+
+The following illustration showcases the outline of the blueprint solution.
+
+![Flowchart for the bot connector solution](images/bot-wm-aws.png "Flowchart for the bot connector solution")
+
+## Contents
 * [Solution components](#solution_components "Goes to the Solutions components section")
 * [Prerequisites](#prerequisites "Goes to the Prerequisites section")
 * [Implementation steps](#implementation_steps "Goes to the Implementation steps section")
 * [Additional resources](#additional_resources "Goes to the Additional resources section")
 
 ## Solution components
 
-[comment]: # List components in logical order (for example, alphabetical, order of use in the solution, Genesys vs. third-party components, large components followed by smaller ones) Copy description from library, if description exists. Otherwise, follow format and brevity of descriptions in library. Format and examples follow. &&: we need to put library in public space.
-
-* **Genesys Cloud Bot Connector** - A set of RESTful APIs that allows you to connect to any bot platform to send and receive utterances. Available in beta and for digital messaging channels only. Please contact your Genesys Account or Partner Manager to enable this feature
-* **Microsoft Azure account** - To be able to create the Power VA bot
-* **AWS account** - To host the Lambda function that is acting as the Bot Interpreter application that is written in Python.
-* **REST client** - Something to submit a HTTP PUT with JSON payload to the Bot Connector API to create the botList. Postman, YARC or any application will work.
-* **A website to deploy Web Messaging** - To be able to test the final result.
+* **Genesys Bot Connector** - The Genesys Bot Connector configuration allows your third-party bots to interact conversationally with customers. A set of RESTful APIs that allows you to connect to any bot platform to send and receive utterances.
```

Comment:
Second sentence is a fragment. Start with, "In this solution, the Genesys Bot Connector provides a set of..."